File: benchmarks/reference/java/FibStress.java
/** FibStress.java — Recursive fibonacci(35) benchmark.
    Equivalent to examples/il/benchmarks/fib_stress.il */
public class FibStress {
    static long fib(long n) {
        if (n <= 1) return n;
        return fib(n - 1) + fib(n - 2);
    }

    public static void main(String[] args) {
        long result = fib(35);
        System.exit((int)(result & 0xFF));
    }
}

File: benchmarks/reference/csharp/FibStress.cs
/* fib_stress — Recursive fibonacci(35) benchmark.
   Equivalent to examples/il/benchmarks/fib_stress.il */
using System;

class FibStress
{
    static long Fib(long n)
    {
        if (n <= 1) return n;
        return Fib(n - 1) + Fib(n - 2);
    }

    static int Main()
    {
        long result = Fib(35);
        return (int)(result & 0xFF);
    }
}
